float64 enu_x
float64 enu_y
float64 enu_z
float64 latitude
float64 longitude
float64 altitude
bool flag_Est
bool flag_EstRaw
